target/linux/ramips/dts/mt7621_adslr_g7.dts

```diff
@@ -64,23 +64,28 @@
 			};
 
 			factory: partition@40000 {
-				compatible = "nvmem-cells";
```

Comment:
This compatible string may still necessary. Not sure.

ref:
https://github.com/torvalds/linux/blob/305230142ae0637213bf6e04f6d9f10bbcb74af8/Documentation/devicetree/bindings/mtd/partitions/nvmem-cells.yaml#L10-L11

Reply:
wrong file: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/Documentation/devicetree/bindings/nvmem/nvmem.yaml?h=v6.6.1

Reply:
It is worth noting that @rmilecki also retains this string in daaa0c1b250498acbf91697dfc75c9d3fbac3596.

Reply:
works fine on my archer with the other PR which also lacks nvmem-cells.

Reply:
@DragonBluep I put/left that by mistake, sorry, I'll have to fix it one day.